Blackfynn/data-portal
# Download files #1a20bv

# Description

The purpose of this PR is to allow users to download files.

## Type of change


# How Has This Been Tested?

- Go to the browse page and search for files
- Click on "Download File" button for various files of various size
- Files should download.

# Checklist:
